### PR TITLE
[doc-update]: ROSA Hosted Wrapper

### PR DESCRIPTION
**Pull request by krishvoor**
This PR documents all parameters used in the rosa-hosted-wrapper
the script, additionally also provides sample usage.

### CODE REVIEW

**Comment by krishvoor** [comment]
Hi 👋 request for review

**Review by morenod — APPROVED** [review]
LGTM

File: rosa-hypershift/README.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+| --es-insecure | If ES is setup with ssl, but can disable tls cert verification | False |
+| --es-index | The index to write to. | hypershift-wrapper-timers |
+| --es-index-retry | Number of retries to connect to ES | 5 |
+| --es-index-only | Upload all metadata.json files found under PATH to elasticsearch | -- |
+
+## Optional variables:
+
+Optional variables are parameters that offer flexibility and allow users to customize the behavior or configuration of the program based on their specific needs.
+
+| Option | Description | Default |
```
Comment (vishnuchalla) [inline rosa-hypershift/README.md:85]: Now that we have these many arguments being passed in the CLI, instead can we simply use configuration files to do that? Open to other thoughts too.

Comment (krishvoor) [reply rosa-hypershift/README.md:85]: > Now that we have these many arguments being passed in the CLI, instead can we simply use configuration files to do that? Open to other thoughts too.

Yes, some are important and the rest are "optional" parameters, would be good to capture all values into a default conf. file and supply the rest via cli

But should be tackled outside the scope of this PR thinking via an RFE.

**Comment by krishvoor** [comment]
Hi 👋 @mukrishn @dry923 @smalleni @vishnuchalla request for review.

**Review by vishnuchalla — APPROVED** [review]

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**